### ISSUE TITLE
Provide an overview of goals, scope and background of the Marketing & Outreach Working Group

### ISSUE THREAD

**Issue by claredillon** [InnerSourceCommons/InnerSourceMarketing#10]

**Comment by claredillon** [comment]
Current working document here: https://docs.google.com/document/d/1WP2jf0RSkWiwIaE1LuCA5TQOJopZjoAs2kEADgz6hGM/edit?usp=sharing

Moving to GitHub to track evolution.

**Comment by lenucksi** [comment]
Dependent upon #19.

**Comment by spier** [comment]
@claredillon I assume that you are working on this? 
Feel free to unassign yourself, or even move it back to TODO if you are not actually planning to work on this before the next session.

**Comment by claredillon** [comment]
Updated this to link to the GitHib project board for an up-to-date view of activities. This version is now ready for review. Thanks @spier for already commenting. Because I believe it it is an accurate reflection of our discussions and intent so far, I will consider that done if one more member of the WG reviews and agrees.

**Comment by spier** [comment]
The title of this card is "Define scope of the Marketing & Outreach Working Group" which I read roughly like "define the mission statement". However the title of the doc is "Marketing & Outreach Working Group Overview", which seems to be much broader.

Therefore I am not 100% sure which part of the document needs to be reviewed, in order to finish this work and close this issue.

@claredillon can you help?

**Comment by claredillon** [comment]
Edited the title to reflect what's included in document. 
Assuming that if it is a current accurate reflection of our goals, scope and background/origin to help people understand the purpose of the working group and the initial thoughts we had on how it should be operated. 

Added reference to the Working Agreement document for more details.  Question: should we move the points on operational principles there?

Added link to marketing metrics report (which is updated monthly). 

Everyone - if there is something else we should elaborate on in the doc - let me know, otherwise, if it is an accurate description of what we have decided so far, propose we close and create new card to review and update in say, 3 months.
